portfolio/src/main/webapp/index.html

```diff
@@ -39,13 +39,20 @@ <h1>Chidindu Alim's Portfolio </h1>
         <p>Click below to get an insightful message:</p>
         <button onclick="addRandomGreeting()">What are you today?</button>
         <div id="greeting-container"></div>
-        <h2>Server Date</h2>
+        <p>Click below to get the artist of the day:</p>
+        <button onclick="showWriters()">Today's writer is......</button>
+        <div id="random-writer-container"></div>
+        <form action="/form-handler" method="POST">
+            <p>Please leave a message for me below:</p>
+            <textarea name="text-input">Hi! I'm happy you're here😁</textarea>
```

Comment:
Probably better to use `placeholder` attribute here than having the text prepopulate the textarea. 
https://www.w3schools.com/tags/att_textarea_placeholder.asp